backend/de.metas.business/src/test/java/de/metas/project/workorder/conflicts/ResourceAllocations_TestCase_ActualAndSimulation_Test.java

```diff
@@ -1,50 +1,37 @@
 package de.metas.project.workorder.conflicts;
 
+import au.com.origin.snapshots.Expect;
+import au.com.origin.snapshots.junit5.SnapshotExtension;
 import com.google.common.collect.ImmutableList;
 import de.metas.calendar.simulation.SimulationPlanId;
 import de.metas.calendar.util.CalendarDateRange;
 import de.metas.product.ResourceId;
 import de.metas.project.workorder.resource.WOProjectResourceId;
-import de.metas.test.SnapshotFunctionFactory;
-import org.adempiere.test.AdempiereTestHelper;
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 
 import javax.annotation.Nullable;
 import java.time.Instant;
 import java.time.LocalDate;
 import java.time.ZoneId;
 import java.time.temporal.ChronoUnit;
 
-import static io.github.jsonSnapshot.SnapshotMatcher.expect;
-import static io.github.jsonSnapshot.SnapshotMatcher.start;
-import static io.github.jsonSnapshot.SnapshotMatcher.validateSnapshots;
 import static org.assertj.core.api.Assertions.*;
 
+@ExtendWith(SnapshotExtension.class)
 public class ResourceAllocations_TestCase_ActualAndSimulation_Test
 {
-	@BeforeAll
-	static void beforeAll()
-	{
-		start(AdempiereTestHelper.SNAPSHOT_CONFIG, SnapshotFunctionFactory.newFunction());
```

Comment:
@mrie04 @metas-ts is there any place where we config that config and SnapshotFunctionFactory.newFunction() to be used?

Reply:
in the new snapshot implementation i use snapshot.properties config files for configuration ( serializer.orderedJson=au.com.origin.snapshots.jackson.serializers.DeterministicJacksonSnapshotSerializer ). It doesn't use SnapshotFunctionFactory atm